app/src/renderer/js/main.js

```diff
@@ -342,6 +346,13 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   };
 
+  aspectRatioSelector.onchange = handleSizeChange;
```

Comment:
Prefer `addEventListener` for all the event handlers.

https://stackoverflow.com/a/35093997/64949

Reply:
We're using `.onchange`, `.onclick` everywhere else. But I agree we should them too later on.

Reply:
Yes, but better to do the right thing for new code.

Reply:
This should really be a linting rule: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/117